packages/orbit-components/src/ErrorFormTooltip/Tooltip/index.tsx

```diff
@@ -87,28 +60,23 @@ const ErrorFormTooltip = ({
     return () => {
       window.removeEventListener("keydown", handleTab);
     };
-  }, [onShown, isHelp, helpClosable]);
+  }, [onShown, isHelp, helpClosable, refs.floating]);
 
-  const isVertical =
-    attrs.popper &&
-    (attrs.popper["data-popper-placement"] === "top-start" ||
-      attrs.popper["data-popper-placement"] === "top-end")
-      ? "bottom"
-      : "top";
+  const isVertical = placement === "top-start" || placement === "top-end" ? "bottom" : "top";
```

Comment:
Yeah, I got the same question as you. I think you are right, because we don't have any auto-positioning in place, but I did not want to risk the logic here while I was implementing the migration. Giving it a second thought now, I guess we can remove and come back if some problem is noticed or reported